src/commands/workspace/logs.ts

```diff
@@ -47,21 +48,23 @@ export default class Logs extends Command {
     const cheTasks = new CheTasks(flags)
     const k8sTasks = new K8sTasks()
 
+    const follow = flags.follow
```

Comment:
it's personal taste. But I would not declare here const and use directly flags.follow as before.